pkg/option/config.go

```diff
@@ -2271,7 +2271,6 @@ var (
 		EnableIPv4:                   defaults.EnableIPv4,
 		EnableIPv6:                   defaults.EnableIPv6,
 		EnableIPv6NDP:                defaults.EnableIPv6NDP,
-		EnableSCTP:                   defaults.EnableSCTP,
```

Comment:
Looks unrelated to this PR, was this committed by mistake?

Reply:
🤔

Reply:
strange, yes it seems so.

Reply:
This is still present in the PR.

Reply:
woops, fixed/rebased into wrong commit, trying again

Reply:
ok that should do it.